### PR TITLE
Redirect users filing kubectl issues to the kubernetes/kubectl repo

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 **Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.):
 
+**Note:** Please file issues for subcomponents under the appropriate repo
+
+| Component | Repo                                                               |
+| --------- | ------------------------------------------------------------------ |
+| kubectl   | [kubernetes/kubectl](https://github.com/kubernetes/kubectl/issues) |
```
Comment (xiangpengzhao) [inline .github/ISSUE_TEMPLATE.md]: It would be great to also provide an issue template in kubectl repo. And link this to /issues/new might be better?

**Comment by pwittrock** [comment]
Done.  PTAL

**Pull request merged**